save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -200,8 +206,27 @@ class DockerService(private val configProperties: ConfigProperties) {
         unzipArchivesAmongAdditionalFiles(execution, isStandardMode, testSuitesDir, resourcesPath)
 
         val saveCliExecFlags = if (isStandardMode) {
+            val execCmd = execution.execCmd
+            val batchSizeForAnalyzer = execution.batchSizeForAnalyzer
             // create stub toml config in aim to execute all test suites directories from `testSuitesDir`
-            testSuitesDir.resolve("save.toml").apply { createNewFile() }.writeText("[general]")
+            val exeCmdForTomlConfig = if (execCmd.isNullOrBlank()) "" else "execCmd =  \"$execCmd\""
```

Comment:
This is where the serialization is needed… 
I will need to think of changing and serializing fields partially… 
https://github.com/akuleshov7/ktoml/pull/90

Reply:
Added fixme for now